docs/getting-started.adoc

```diff
@@ -1,4 +1,94 @@
-== Getting started
+== Getting Started with AeroGear Services JavaScript SDK
 
-SDK is under construction. 
-No SDK available at the moment.
+This guide provides general information for creating applications that utilize the AeroGear Services JavaScript SDK.
+
+== Prerequisites for Apache Cordova
+
+In order to develop a Cordova application utilizing the SDK you will need the following prerequisites installed:
+
+ * The link:https://nodejs.org/en/download/[Node.js] runtime and NPM
+ * https://github.com/aerogear/mobile-cli[AeroGear Mobile CLI]
+ * Apache link:https://cordova.apache.org/docs/en/latest/guide/cli/#installing-the-cordova-cli[Cordova CLI]
+
+== Creating an Apache Cordova application
+
+After installing the prerequisites, refer to the link:https://cordova.apache.org/docs/en/latest/guide/cli/index.html#create-the-app[App Creation] guide from Apache Cordova to create your first application.
+
+== Installing the AeroGear JavaScript SDK
```

Comment:
Since this is a getting-started, what do you think about avoiding as much verbosity as possible? This paragraph doesn't add much, IMO. We could leave the next one with the relevant lines of code, that is `cordova plugin add @aerogearservices/core-cordova`